settings.py

```diff
@@ -25,4 +25,8 @@
 
 INSTALLED_APPS = [
     "db",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
 ]
+
+AUTH_USER_MODEL = "db.User"
```

Comment:
The `AUTH_USER_MODEL` is set to `db.User`, which is necessary when using a custom user model in Django.